basic_samples/Authentication/ClientCredentialFlow/DotNet/ClientCredentialFlow/ClientCredentialFlow.csproj

```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
-    <Version>1.1.1</Version>
```

Comment:
dropping version from app and just in documentation?  thats consistent so probably fine